### PR TITLE
Add option to disable providers

### PR DESCRIPTION
**Pull request by djbulsink**
Add options `providers.kubernetesCRD.enabled` and `providers.kubernetesIngress.enabled` to enable and disable providers. 

Default both providers are enabled.

### CODE REVIEW

File: traefik/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 8.2.1
+version: 8.2.2
```
Comment (SantoDE) [inline traefik/Chart.yaml:3]: As this PR is adding new configuration, it should rather be a minor enhancement then a patch version

Comment (djbulsink) [reply traefik/Chart.yaml]: I updated the version to 8.5.0

**Comment by djbulsink** [comment]
Hi @SantoDE , 

There is a high change of conflict in version number in the Chart,yaml, when there are multiple pull requests open and merged at the same time. I was wondering if it would be possible to change the version in Chart.yaml after the PRs are merged? 

I also wanted to know if I have to add a label or do some other actions after I resolved the issues on a PR? 

Best regards, 

Dirk-Jan

**Comment by SantoDE** [comment]
Hello @djbulsink,

sorry that the merge failed. Can you please allow maintainers to do edits on this PR?

**Comment by djbulsink** [comment]
@SantoDE ,

No problem. I changed the version number. The option "Allow edits by maintainers" was allready enabled,

**Pull request merged**